artifacts/pom.xml

```diff
@@ -208,14 +208,24 @@
 
 		<!-- certify sunbird plugin -->
 		<certify-plugin.location>/usr/share/nginx/html/artifactory/libs-release-local/certify/certify-plugin</certify-plugin.location>
-		<certify-sunbird-plugin.version>0.2.0-SNAPSHOT</certify-sunbird-plugin.version>
+		<certify-sunbird-plugin.version>0.2.1-SNAPSHOT</certify-sunbird-plugin.version>
 		<certify-sunbird-plugin.fileName>certify-sunbird-plugin.jar</certify-sunbird-plugin.fileName>
+
+		<!-- certify mosip identity plugin -->
+		<certify-plugin.location>/usr/share/nginx/html/artifactory/libs-release-local/certify/certify-plugin</certify-plugin.location>
+		<certify-mosip-identity-plugin.version>0.2.1-SNAPSHOT</certify-mosip-identity-plugin.version>
+		<certify-mosip-identity-plugin.fileName>certify-mosip-identity-plugin.jar</certify-mosip-identity-plugin.fileName>
+
+		<!-- certify mock plugin -->
+		<certify-plugin.location>/usr/share/nginx/html/artifactory/libs-release-local/certify/certify-plugin</certify-plugin.location>
+		<certify-mock-plugin.version>0.2.1-SNAPSHOT</certify-mock-plugin.version>
+		<certify-mock-plugin.fileName>certify-mock-identity-plugin.jar</certify-mock-plugin.fileName>
 		
 		<!-- esignet IDA wrapper -->
 		<esignet-ida-wrapper.version>1.2.1-SNAPSHOT</esignet-ida-wrapper.version>
 		<esignet-ida-wrapper.fileName>esignet-ida-wrapper.jar</esignet-ida-wrapper.fileName>
-		<esignet-digital-credential-wrapper.version>0.1.0-SNAPSHOT</esignet-digital-credential-wrapper.version>
-                <esignet-digital-credential-wrapper.fileName>sunbird-rc-esignet-integration-impl.jar</esignet-digital-credential-wrapper.fileName>
+		<esignet-digital-credential-wrapper.version>0.2.0-SNAPSHOT</esignet-digital-credential-wrapper.version>
```

Comment:
can you change this also to 0.2.1-snapshot, since its part of digital credentials plugin module

Reply:
Changed version to 0.2.1-snapshot for esignet-wrapper